test262/per-file/index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>LibJS test262 per-file results</title>
+    <link rel="shortcut icon" type="image/png" href="/favicon.png" />
+    <link rel="stylesheet" href="../main.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@700&family=Roboto:wght@400;700&display=swap"
+    />
+    <style></style>
+  </head>
+
+  <body>
+    <main>
+      <h1>LibJS test262 per-file result viewer</h1>
+      <p>
```

Comment:
not sure if we really need to repeat all of this tbh

Reply:
I removed all the text at the top, it was the same stuff as the main page anyway.